### ISSUE TITLE
Snooze all option - Feature Request

### ISSUE THREAD

**Issue by loopy321** [JohanDegraeve/xdripswift#368]
It would be great to have the "Snooze All" option to have some peace after doing a correction and waiting for the effects.

Or something like the old Spike Snooze selectable pop-up:
![Spike snoozer ](https://user-images.githubusercontent.com/1026345/183124333-37a5b190-42d5-4649-87c1-9bf4416184ae.png)

**Comment by bjornoleh** [comment]
Yes please, and perhaps an easier way to disable/enable all alarms. If more than one alarm type is defines, it takes a lot of clicks to disable them all. (We are running two instances of xdripswift with overlapping sensors sessions, and need to switch alarms from one instance to the other)

**Comment by paulplant** [comment]
Yes... a global "Snooze All" sounds like a good option. I think we can probably add this to the list 👍

**Comment by paulplant** [comment]
Snooze All is available in the current app, and the snooze screen has also been updated in 7.0.0. You can now pause alerts together without changing each alert individually. We’ll close this as implemented. Please open a new issue if you run into any problems.
